ConcurrencyHomework/src/concurrency/Person.java

```diff
@@ -0,0 +1,46 @@
+package concurrency;
+
+import java.time.LocalDate;
+
+public class Person {
+
+    private String name;
+    private LocalDate date;
+    private Gender gender;
+
+    public Person(String name, LocalDate date, String gender) {
+        this.name = name;
+        this.date = date;
+        if (gender.equals("male"))
```

Comment:
Logica de interpretare a gender-ului ar fi trebuit incapsulata in enum-ul `Gender ` intro metoda scrisa de tine,